packages/playwright-core/src/server/isomorphic/utilityScriptSerializers.ts

```diff
@@ -25,7 +27,8 @@ export type SerializedValue =
     { a: SerializedValue[], id: number } |
     { o: { k: string, v: SerializedValue }[], id: number } |
     { ref: number } |
-    { h: number };
+    { h: number } |
+    { ta: { b: string, k: TypedArrayKind } };
```

Comment:
Sounds like we need a bunch of page.evaluate tests for this.

Reply:
Landing this w/o coverage does not feel right, probably supporting the protocol serializer and testing e2e is the right approach.

Reply:
I've added support to the protocol serializer and a bunch of page.evaluate tests.